OsuPlayer.Tests/OsuPlayer.Network/PasswordManagerTests.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Threading.Tasks;
+using NUnit.Framework;
+using OsuPlayer.Network.Security;
+
+namespace OsuPlayer.Tests.OsuPlayer.Network;
+
+public class PasswordManagerTests
+{
+    [Test]
+    public async Task PasswordManagerTest()
+    {
+        var r1 = PasswordManager.CheckIfPasswordMeetsRequirements("");
+        var r2 = PasswordManager.CheckIfPasswordMeetsRequirements("Test");
+        var r3 = PasswordManager.CheckIfPasswordMeetsRequirements("Test123!_?");
+        
+        Assert.IsFalse(r1);
+        Assert.IsFalse(r2);
+        Assert.IsTrue(r3);
+    }
```

Comment:
Please remove the async/Task on this test as it does not use any awaits. Also split into different test cases, as already stated earlier.

Reply:
oops my bad shouldn't be there tbh. Probably a copy paste error